infra/k8s/clean.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+START_TIME=`date +%s`
+
+echo "Resetting Testground environment..."
+echo
+
+kubectl delete pods -l app=testground --force --grace-period=0
```

Comment:
@hacdias that could be the initial health/repair functionality for Kubernetes:
1. Count the number of worker nodes in the cluster at the time of the call to the command (see `maxPods` in cluster_k8s.go)
2. Delete all pods labeled with `app=testground`
3. Delete all sidecar pods
4. Wait for pods with label `app=testground` to be 0 (all states, including Terminated)
5. Wait for worker nodes number of sidecar pods to appear.

---

This doesn't account for the race of somebody running another `testground run` command, while we are terminating, but I'd say this functionality is not something we need right now :)